src/DotNetBridge/DotNetBridge.csproj

```diff
@@ -9,7 +9,7 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputPath>..\$(Platform)\$(Configuration)\</OutputPath>
     <Configurations>DbgWinPy3.6;DbgWinPy3.5;DbgWinPy2.7;RlsWinPy3.6;RlsWinPy3.5;RlsWinPy2.7;DbgLinPy3.6;DbgLinPy3.5;DbgLinPy2.7;RlsLinPy3.6;RlsLinPy3.5;RlsLinPy2.7;RlsMacPy3.6</Configurations>
-    <Version>0.6.0</Version>
+    <Version>0.9.0</Version>
```

Comment:
Let's change this to 0.10

Reply:
Changed

---
In reply to: [247268196](https://github.com/Microsoft/NimbusML/pull/77#discussion_r247268196) [](ancestors = 247268196)